samples/zigbee/ncp/sample.yaml

```diff
@@ -0,0 +1,21 @@
+sample:
+  name: Zigbee NCP
+  description: Zigbee network co-processor
+
+tests:
+  zigbee.ncp:
+    build_only: true
+    platform_allow: nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf52840dongle_nrf52840 nrf21540dk_nrf52840
```

Comment:
I'm not an expert in `sample.yaml`, but should we include `nrf52840dongle_nrf52840` and  `nrf21540dk_nrf52840` platforms here?

Reply:
you're right, nrf52840dongle_nrf52840 can't use uart transport. Why nrf21540dk_nrf52840 shouldn't be on the list?